tests/ga/test_monitor.py

```diff
@@ -30,6 +34,13 @@
 from tests.tools import *
 
 
+_MSG_INFO = "This is our test info logging message {0} {1}"
+_MSG_WARN = "This is our test warn logging message {0} {1}"
+_MSG_ERROR = "This is our test error logging message {0} {1}"
+_MSG_VERBOSE = "This is our test verbose logging message {0} {1}"
+_DATA = ["arg1", "arg2"]
+
```

Comment:
This is not used anymore. Will remove this code. More exhaustive testing of the periodic_* is done in test_logger.